validator/tests/test_validation.py

```diff
@@ -568,6 +569,7 @@ def test_setup_filtering_max(self):
         start_time = time.time()
 
         for dataset in Dataset.objects.all():
+            print(dataset.pretty_name)
```

Comment:
The print statements in the test_setup_filtering_max test should probably be replaced by log statements, a la
`self.__logger.info(dataset.pretty_name)`
At least this makes it much easier to search for test output in valentina.log on my system.